src/sentry/search/eap/spans/filter_aliases.py

```diff
@@ -0,0 +1,219 @@
+from typing import Literal
+
+from sentry.api.event_search import SearchFilter, SearchKey, SearchValue
+from sentry.exceptions import InvalidSearchQuery
+from sentry.models.release import Release
+from sentry.models.releases.util import SemverFilter
+from sentry.search.events import constants
+from sentry.search.events.filter import (
+    _flip_field_sort,
+    handle_operator_negation,
+    parse_semver,
+    to_list,
+)
+from sentry.search.events.types import SnubaParams
+from sentry.search.utils import parse_release, validate_snuba_array_parameter
+
+
+def release_filter_converter(params: SnubaParams, search_filter: SearchFilter) -> SearchFilter:
+    if search_filter.value.is_wildcard():
+        operator = search_filter.operator
+        value = search_filter.value
+    else:
+        operator_conversions = {"=": "IN", "!=": "NOT IN"}
+        operator = operator_conversions.get(search_filter.operator, search_filter.operator)
+
+        if params.environments:
+            environments = [env for env in params.environments if env is not None]
+        else:
+            environments = []
+
+        value = SearchValue(
+            [
+                part
+                for v in to_list(search_filter.value.value)
+                for part in parse_release(
```

Comment:
nit: hmmmmm double list comprehension 😨

Reply:
gonna leave this alone so it doesnt deviate from the source just yet